packages/askui-nodejs/src/utils/proxy/proxy-utils.ts

```diff
@@ -0,0 +1,111 @@
+import fs from 'fs';
```

Comment:
Can we mark this file somewhat as belonging to the test? It looks like actual source code (not only to the reader but also to our tooling, e.g., eslint, typescript...

Reply:
This file should be a helper functions for testing proxies. But I don't know which file ending I can use to mark this as an helper file only for tests

Reply:
But then is jest complaining and wants to have min. one test suite